.gitignore

```diff
@@ -2,3 +2,4 @@
 _book
 l10ndocs-linter
 node_modules
+package-lock.json
```

Comment:
`package-lock.json` should be committed to the repository. Please revert this line.